### ISSUE TITLE
fcma: Document normalization behavior

### ISSUE THREAD

**Issue by mihaic** [brainiak/brainiak#388]
Follow-up on discussion from Gitter chat room:
https://gitter.im/brainiak/brainiak?at=5bc9d5a564cfc273f9f03ad6

CC @amennen, @qihongl, @CameronTEllis, @manojneuro.

**Comment by amennen** [comment]
Actually, I just tested the same thing on my leave-one-subject-out classification. For all 20 classifiers for each of the 20 subjects left out, when tested on the left out subject, the mean of the decision function values is always equal to the intercept of the decision function. So this seems to be a broader property of FCMA normalization.

**Comment by amennen** [comment]
This is the definition of the optimal intercept of the hyperplane (the average of the two worst examples from each class):

![image](https://user-images.githubusercontent.com/16296896/47389992-3aab6a00-d6e4-11e8-8d0d-9919b798aff6.png)

**Comment by qihongl** [comment]
> Actually, I just tested the same thing on my leave-one-subject-out classification. For all 20 classifiers for each of the 20 subjects left out, when tested on the left out subject, the mean of the decision function values is always equal to the intercept of the decision function. So this seems to be a broader property of FCMA normalization.

Yeah, assuming the data is normalized (centered and rescaled) + each run contribute the roughly the same cost term (related to the normalization assumption), indeed I can convince myself that this is a property of the fitting procedure.

**Comment by amennen** [comment]
Okay, so with Manoj's help (thank you Manoj!), we concluded that if the distance to the hyperplane is `( w^T*x + b)`, then when the data is zscored, the mean of all distances must = b because the mean of x is 0. So this means that any testing data that is zscored will have the same mean distance equal to b.

Moving forward, I don't think fcma should refer to the decision function value as "confidence" in its classifier class [here](https://github.com/brainiak/brainiak/blob/master/brainiak/fcma/classifier.py), as it is not the confidence, but instead scales with the points that it's tested with. We should include calculations of probability estimates (setting probability=true) (for mvpa/fcma) so that users can get continuous probability values that won't have a mean that is the same across people/runs.

**Comment by yidawang** [comment]
Thanks for investigating this, @amennen 

Within the same set of test samples, can the distance value indicate the confidence?

**Comment by manojneuro** [comment]
@yidawang for each individual sample, distance can indicate confidence. One alternative is to get a distribution of distances for each predicted label. I have mentioned this to @amennen and she is looking into explloring this option along with computing probabilities.
